utils/minor-release.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Script: minor-release.sh
+# Description:
+#   Cherry-picks a list of commits, amends each with the original commit hash for tracking,
+#   and generates a summary from the short github commit messages with links to each commit.
+#
+# Usage:
+#   ./minor-release.sh <commit1> <commit2> ...
+#
+# Example:
+#   ./minor-release.sh abc1234 def5678
+
+set -e
+
+# Check if at least two arguments are provided (repo and at least one commit)
+if [ "$#" -lt 1 ]; then
+    echo "Usage: $0 <commit1> <commit2> ..."
+    echo "Example: $0 abc1234 def5678"
+    exit 1
+fi
+
+# The first argument is the GitHub repo identifier
```

Comment:
This comment seems outdated since the repo name is hardcoded? (also the one above that mentions that least two arguments are needed)

Reply:
True!
```suggestion
```